data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -452,4 +456,5 @@
     field_names.ELIGIBLE_FUDS_BINARY_FIELD_NAME,
     field_names.AML_BOOLEAN,
     field_names.HISTORIC_REDLINING_SCORE_EXCEEDED,
+    field_names.PERCENT_OF_TRIBAL_AREA_IN_TRACT
```

Comment:
Did we confirm that @KameronKerger and @vim-usds are comfortable with just two digits on the percent of Tribal area in tract?

Reply:
We did talk about it and that was the agreement

Reply:
Following up to @lucasmbrown-usds 's question. I believe @mattbowen-usds and I have agreed to this API contract:

Let's assume the selected tract has a tribal percentage is `45%`. Assumption here is the value that is sent to the FE is `0.45`. 

In another tract, let's assume the tribal area percentage is `1%`. This will this be a sent as `0.01`. 

For percents _less than_ 1%, ie, `.1%` the BE signal will have a value of `0` or `0.00`. In this case the FE will show this as 'less than 1%'. 

For percents that are _equal to_ `0%`, the BE signal will be `null` and the FE will show 0%.

If this is all accurate, it makes me conclude that two digits of accuracy will suffice.